### PR TITLE
Make this work again

### PR DESCRIPTION
**Pull request by languitar**
This a single batch of stuff that updates the project to work again and to comply to the current conventions.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+#
+# ============================================================
+
+from setuptools import setup
+
+setup(name='rsb-spread-python',
+      version='1.0.0-dev',
+      description='Spread transport for rsb-python',
+      author='Johannes Wienke',
```
Comment (scymtym) [inline setup.py]: Please add me here.

File: README.md [hunk 2]
```diff
@@ -1 +1,63 @@
-Currently not working correctly. Has to be reenabled after the python 3 port of openrsx/rsb-python.
+[![Build Status](https://travis-ci.org/open-rsx/rsb-spread-python.svg?branch=master)](https://travis-ci.org/open-rsx/rsb-spread-python) [![Coverage Report](https://codecov.io/gh/open-rsx/rsb-spread-python/branch/master/graph/badge.svg)](https://codecov.io/gh/open-rsx/rsb-spread-python)
+
+# Introduction
+
+This repository contains the Python implementation of the [spread] based transport for the [Robotics Service Bus](https://github.com/open-rsx) middleware.
+
+**The full documentation for RSB can be found at <https://docs.cor-lab.de/manual/rsb-trunk/index.html>.**
```
Comment (scymtym) [inline README.md:7]: Link to the new documentation?

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      author_email='jwienke@techfak.uni-bielefeld.de',
+      license='LGPLv3+',
+      url='https://code.cor-lab.org/projects/rsb',
```
Comment (scymtym) [inline setup.py]: Update URL.

**Review by scymtym — CHANGES_REQUESTED** [review]
A few comments, otherwise LGTM.

File: setup.py (hunk 1 above)
Comment (languitar) [reply setup.py]: There doesn't seem to be actual support for multiple authors or maintainers. Leaving out both is also not supported.

File: README.md (hunk 2 above)
Comment (languitar) [reply README.md:7]: has to be changed in all readme. I would try to do that in a separate batch.

File: setup.py (hunk 1 above)
Comment (languitar) [reply setup.py]: Let's see whether a comma is ok

**Review by scymtym — APPROVED** [review]

**Comment by languitar** [comment]
are you going to merge this or should I?

**Comment by scymtym** [comment]
on it

**Pull request merged**